engine/src/main/java/org/terasology/logic/players/CameraClientSystem.java

```diff
@@ -65,15 +98,38 @@ public void resetCamera() {
     }
 
```

Comment:
Also: why do you destroy the camera to reset its position, rather than, say, just change its position to some kind of default?